terracumber_config/tf_files/local_mirror.tf

```diff
@@ -86,17 +86,18 @@ terraform {
   required_providers {
     libvirt = {
       source = "dmacvicar/libvirt"
-      version = "0.8.1"
+      version = "0.6.3"
     }
   }
 }
 
+
 provider "libvirt" {
-  uri = "qemu+tcp://grog.mgr.prv.suse.net/system"
+  uri = "qemu+tcp://suma-05.mgr.suse.de/system"
 }
 
 locals {
-  pool = "ssd"
+  pool = "mnoel_disks"
```

Comment:
I wouldn't use a personal disk for this terraform file, unless it's clear on its filename.

Reply:
done